filters/include/pcl/filters/impl/convolution.hpp

```diff
@@ -239,164 +240,40 @@ namespace pcl
   namespace filters
   {
     template<> pcl::PointXYZRGB
-    Convolution<pcl::PointXYZRGB, pcl::PointXYZRGB>::convolveOneRowDense (int i, int j)
```

Comment:
Can you not remove this header entirely? Do the declarations still need to reside here?

Reply:
Yes. The declaration is needed to be visible (basically, this is just a normal function now)